src/forms/Household.js

```diff
@@ -219,12 +220,13 @@ const MemberField = function ({ household, time, setHousehold, setClientProperty
       </Columns.Two>
 
       <Columns.Three>
-        <Input
-          className = {time + '-member-age ' + time}
-          onChange  = {onMemberChange}
+        <ManagedNumberField
           value     = {member.m_age}
           name      = {'m_age'}
-          type      = {'number'} step = {'1'} min = {'0'} />
```

Comment:
I would suggest keeping `step` and `min` and changing `<ManagedNumberInput>` to pass through extra props like these.

Reply:
There are no arrow buttons, so currently no point in `step` and `min`, since those attributes don't control anything about what happens when people type in values. I'm not sure it's worth future proofing these.

Reply:
Some browsers will add an alternative way of interacting with number inputs. I know in Chrome you can use the arrow keys to step the value up and down.

Minor though.

Reply:
Ah, I didn't even consider arrow keys. That's a good point then. This should definitely go on the 'small improvements' list!